From 0b597df99bb17e60c221b29c2f703e01ddd883e1 Mon Sep 17 00:00:00 2001
From: Sandie Cao <sandie.cao@deepcomputing.io>
Date: Fri, 27 Sep 2024 14:28:13 +0800
Subject: [PATCH 30/46] riscv: defconfig: add starfive drm and hdmi config

Add CONFIG_DRM_VERISILICON_DC8200 for starfive jh7110 DRM,
and add CONFIG_DRM_INNO_STARFIVE_HDMI for starfive hdmi.

Signed-off-by: Sandie Cao <sandie.cao@deepcomputing.io>
---
 arch/riscv/configs/defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/riscv/configs/defconfig b/arch/riscv/configs/defconfig
index 29975b16ab9d..0cd45d6e16c9 100644
--- a/arch/riscv/configs/defconfig
+++ b/arch/riscv/configs/defconfig
@@ -200,6 +200,10 @@ CONFIG_DRM_RADEON=m
 CONFIG_DRM_NOUVEAU=m
 CONFIG_DRM_SUN4I=m
 CONFIG_DRM_VIRTIO_GPU=m
+CONFIG_DRM_INNO_HDMI=m
+CONFIG_DRM_VERISILICON_DC8200=m
+CONFIG_DRM_INNO_STARFIVE_HDMI=y
+CONFIG_DRM_PANEL_ORIENTATION_QUIRKS=m
 CONFIG_FB=y
 CONFIG_SOUND=y
 CONFIG_SND=y
-- 
2.51.0

